create waku@next error on windows

```shell
PS C:\Users\himse> pnpm create waku@next
.../Local/pnpm/store/v3/tmp/dlx-16864    |   +1 +
Packages are hard linked from the content-addressable store to the virtual store.
  Content-addressable store is at: C:\Users\himse\AppData\Local\pnpm\store\v3
  Virtual store is at:             AppData/Local/pnpm/store/v3/tmp/dlx-16864/node_modules/.pnpm
.../Local/pnpm/store/v3/tmp/d.../Local/pnpm/store/v3/tmp/dlx-16864    | Progress: resolved 1, reused 0, downloaded 1, added 1, doneame » waku-project
√ Project Name ... waku-project
√ Choose a starter template » basic-template
Setting up project...

Done. Now run:

cd waku-project
pnpm install
pnpm dev

PS C:\Users\himse> cd .\waku-project\
PS C:\Users\himse\waku-project> pnpm install
Packages: +155
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Packages are hard linked from the content-addressable store to the virtual store.
  Content-addressable store is at: C:\Users\himse\AppData\Local\pnpm\store\v3
  Virtual store is at:             node_modules/.pnpm
Progress: resolved 199, reused 143, downloaded 12, added 155, done
node_modules/.pnpm/@swc+core@1.3.102/node_modules/@swc/core: Running postinstall script, done in 69ms

dependencies:
+ react 18.3.0-canary-c5b937576-20231219
+ react-dom 18.3.0-canary-c5b937576-20231219
+ waku 0.19.0-beta.0

devDependencies:
+ @types/react 18.2.46 (18.2.47 is available)
+ @types/react-dom 18.2.18
+ typescript 5.3.3

Done in 5.4s
PS C:\Users\himse\waku-project> pnpm dev

> waku-project@0.0.0 dev C:\Users\himse\waku-project
> waku dev --with-ssr

ready: Listening on http://localhost:3000/
Failed to resolve dependency: react-server-dom-webpack/client, present in 'optimizeDeps.include'
Failed to resolve dependency: react-server-dom-webpack/client, present in 'optimizeDeps.include'
4:16:25 PM [vite] Error when evaluating SSR module /src/components/Counter.tsx: failed to import "react-server-dom-webpack/server"
|- Error: Cannot find module 'react-server-dom-webpack/server' imported from 'C:/Users/himse/waku-project/src/components/Counter.tsx'
    at nodeImport (file:///C:/Users/himse/waku-project/node_modules/.pnpm/vite@5.0.10/node_modules/vite/dist/node/chunks/dep-R0I0XnyH.js:50537:25)
    at ssrImport (file:///C:/Users/himse/waku-project/node_modules/.pnpm/vite@5.0.10/node_modules/vite/dist/node/chunks/dep-R0I0XnyH.js:50444:30)
    at eval (C:/Users/himse/waku-project/src/components/Counter.tsx:3:50)
    at instantiateModule (file:///C:/Users/himse/waku-project/node_modules/.pnpm/vite@5.0.10/node_modules/vite/dist/node/chunks/dep-R0I0XnyH.js:50506:15)

4:16:25 PM [vite] Error when evaluating SSR module /src/components/App.tsx: failed to import "/src/components/Counter.tsx"
|- Error: Cannot find module 'react-server-dom-webpack/server' imported from 'C:/Users/himse/waku-project/src/components/Counter.tsx'
    at nodeImport (file:///C:/Users/himse/waku-project/node_modules/.pnpm/vite@5.0.10/node_modules/vite/dist/node/chunks/dep-R0I0XnyH.js:50537:25)
    at ssrImport (file:///C:/Users/himse/waku-project/node_modules/.pnpm/vite@5.0.10/node_modules/vite/dist/node/chunks/dep-R0I0XnyH.js:50444:30)
    at eval (C:/Users/himse/waku-project/src/components/Counter.tsx:3:50)
    at instantiateModule (file:///C:/Users/himse/waku-project/node_modules/.pnpm/vite@5.0.10/node_modules/vite/dist/node/chunks/dep-R0I0XnyH.js:50506:15)

Error: Cannot find module 'react-server-dom-webpack/server' imported from 'C:/Users/himse/waku-project/src/components/Counter.tsx'
    at nodeImport (file:///C:/Users/himse/waku-project/node_modules/.pnpm/vite@5.0.10/node_modules/vite/dist/node/chunks/dep-R0I0XnyH.js:50537:25)
    at ssrImport (file:///C:/Users/himse/waku-project/node_modules/.pnpm/vite@5.0.10/node_modules/vite/dist/node/chunks/dep-R0I0XnyH.js:50444:30)
    at eval (C:/Users/himse/waku-project/src/components/Counter.tsx:3:50)
    at instantiateModule (file:///C:/Users/himse/waku-project/node_modules/.pnpm/vite@5.0.10/node_modules/vite/dist/node/chunks/dep-R0I0XnyH.js:50506:15) {
  code: 'ERR_MODULE_NOT_FOUND'
}
Error: Cannot find module 'react-server-dom-webpack/server' imported from 'C:/Users/himse/waku-project/src/components/Counter.tsx'
    at nodeImport (file:///C:/Users/himse/waku-project/node_modules/.pnpm/vite@5.0.10/node_modules/vite/dist/node/chunks/dep-R0I0XnyH.js:50537:25)
    at ssrImport (file:///C:/Users/himse/waku-project/node_modules/.pnpm/vite@5.0.10/node_modules/vite/dist/node/chunks/dep-R0I0XnyH.js:50444:30)
    at eval (C:/Users/himse/waku-project/src/components/Counter.tsx:3:50)
    at instantiateModule (file:///C:/Users/himse/waku-project/node_modules/.pnpm/vite@5.0.10/node_modules/vite/dist/node/chunks/dep-R0I0XnyH.js:50506:15) {
  digest: ''
}
Cannot render RSC Error: Cannot find module 'react-server-dom-webpack/server' imported from 'C:/Users/himse/waku-project/src/components/Counter.tsx'
    at nodeImport (file:///C:/Users/himse/waku-project/node_modules/.pnpm/vite@5.0.10/node_modules/vite/dist/node/chunks/dep-R0I0XnyH.js:50537:25)
    at ssrImport (file:///C:/Users/himse/waku-project/node_modules/.pnpm/vite@5.0.10/node_modules/vite/dist/node/chunks/dep-R0I0XnyH.js:50444:30)
    at eval (C:/Users/himse/waku-project/src/components/Counter.tsx:3:50)
    at instantiateModule (file:///C:/Users/himse/waku-project/node_modules/.pnpm/vite@5.0.10/node_modules/vite/dist/node/chunks/dep-R0I0XnyH.js:50506:15) {
  digest: ''
}
```

--- a/e2e/fixtures/rsc-basic/package.json
+++ b/e2e/fixtures/rsc-basic/package.json
@@ -11,6 +11,7 @@
   "dependencies": {
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "waku": "0.19.0-beta.0"
   },
   "devDependencies": {

--- a/e2e/fixtures/rsc-router/package.json
+++ b/e2e/fixtures/rsc-router/package.json
@@ -11,6 +11,7 @@
   "dependencies": {
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "waku": "0.19.0-beta.0"
   },
   "devDependencies": {

--- a/e2e/fixtures/ssr-basic/package.json
+++ b/e2e/fixtures/ssr-basic/package.json
@@ -11,6 +11,7 @@
   "dependencies": {
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "waku": "0.19.0-beta.0"
   },
   "devDependencies": {

--- a/examples/01_template/package.json
+++ b/examples/01_template/package.json
@@ -11,6 +11,7 @@
   "dependencies": {
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "waku": "0.19.0-beta.0"
   },
   "devDependencies": {

--- a/examples/02_minimal/package.json
+++ b/examples/02_minimal/package.json
@@ -11,6 +11,7 @@
   "dependencies": {
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "waku": "0.19.0-beta.0"
   },
   "devDependencies": {

--- a/examples/03_promise/package.json
+++ b/examples/03_promise/package.json
@@ -11,6 +11,7 @@
   "dependencies": {
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "waku": "0.19.0-beta.0"
   },
   "devDependencies": {

--- a/examples/04_callserver/package.json
+++ b/examples/04_callserver/package.json
@@ -11,6 +11,7 @@
   "dependencies": {
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "waku": "0.19.0-beta.0"
   },
   "devDependencies": {

--- a/examples/05_mutation/package.json
+++ b/examples/05_mutation/package.json
@@ -11,6 +11,7 @@
   "dependencies": {
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "react-wrap-balancer": "1.1.0",
     "waku": "0.19.0-beta.0"
   },

--- a/examples/06_nesting/package.json
+++ b/examples/06_nesting/package.json
@@ -11,6 +11,7 @@
   "dependencies": {
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "waku": "0.19.0-beta.0"
   },
   "devDependencies": {

--- a/examples/07_router/package.json
+++ b/examples/07_router/package.json
@@ -11,6 +11,7 @@
   "dependencies": {
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "waku": "0.19.0-beta.0"
   },
   "devDependencies": {

--- a/examples/08_cookies/package.json
+++ b/examples/08_cookies/package.json
@@ -13,6 +13,7 @@
     "express": "4.18.2",
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "waku": "0.19.0-beta.0"
   },
   "devDependencies": {

--- a/examples/09_cssmodules/package.json
+++ b/examples/09_cssmodules/package.json
@@ -11,6 +11,7 @@
   "dependencies": {
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "waku": "0.19.0-beta.0"
   },
   "devDependencies": {

--- a/examples/10_dynamicroute/package.json
+++ b/examples/10_dynamicroute/package.json
@@ -12,6 +12,7 @@
     "glob": "10.3.10",
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "waku": "0.19.0-beta.0"
   },
   "devDependencies": {

--- a/examples/11_form/package.json
+++ b/examples/11_form/package.json
@@ -11,6 +11,7 @@
   "dependencies": {
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "waku": "0.19.0-beta.0"
   },
   "devDependencies": {

--- a/examples/12_css/package.json
+++ b/examples/12_css/package.json
@@ -14,6 +14,7 @@
     "classnames": "2.3.2",
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "waku": "0.19.0-beta.0"
   },
   "devDependencies": {

--- a/examples/13_path-alias/package.json
+++ b/examples/13_path-alias/package.json
@@ -11,6 +11,7 @@
   "dependencies": {
     "react": "18.3.0-canary-c5b937576-20231219",
     "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "waku": "0.19.0-beta.0"
   },
   "devDependencies": {

--- a/packages/waku/package.json
+++ b/packages/waku/package.json
@@ -73,7 +73,6 @@
     "@swc/core": "1.3.102",
     "@vitejs/plugin-react": "4.2.1",
     "hono": "3.11.12",
-    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219",
     "vite": "5.0.10"
   },
   "devDependencies": {
@@ -81,6 +80,8 @@
     "rollup": "^4.9.2"
   },
   "peerDependencies": {
-    "react": "18.3.0-canary-c5b937576-20231219"
+    "react": "18.3.0-canary-c5b937576-20231219",
+    "react-dom": "18.3.0-canary-c5b937576-20231219",
+    "react-server-dom-webpack": "18.3.0-canary-c5b937576-20231219"
   }
 }

